### ISSUE TITLE
SPAGOBI-1569 - Problem opening 2 olap document

### ISSUE THREAD

**Issue by KnowageServiceAcc** [SpagoBILabs/SpagoBI#1150]
## Details
|Key            |Value                               |
|---------------|------------------------------------|
|**Key**        |SPAGOBI-1569|
|**Type**       |Bug|
|**Status**     |Open|
|**Priority**   |Major|
|**Resolution** ||
|**Affects**    |4.0.0|
|**Fix**        ||
|**Components** |SERVER/OLAP/JPivot|
|**Labels**     ||

## People
|Key         | Value |
|------------|-------|
|**Assignee**|Davide Zerbetto|
|**Reporter**|dan daq|

## Dates
|Key         | Value |
|------------|-------|
|**Created** |2014-01-09T09:23:39.000+0100|
|**Updated** |2014-01-09T09:23:39.000+0100|
|**Resolved**|*n.d.*|
 
## Description
If I open two different Jpivot OLAP documents in two different tabs and I start navigating them, the data of the last document became data of the first one. 
In other words if I switch from a tab to the other and I do an oparation (e.g. drill through) I see the same document!

## Comments
*no comments*
